test/benchmark/tests/utils/report.py

```diff
@@ -160,8 +163,8 @@ def main(s3_bucket_uri):
        f.write(markdownDocument.get_document()) 
 
     # Clean up 
-    run(f"rm -rf /tmp/report")
+    # run(f"rm -rf /tmp/report")
```

Comment:
Is this needed? It might be better to put logic in code such that it can automatically create the report folder if it does not exists so that one does not have to keep commenting/uncommenting out these lines to reruns.

Reply:
Agreed, that's the plan - but will be done later as part of a couple other changes listed here: https://github.com/pytorch/serve/issues/1121. The reason is that the 'wiring' that needs to be done to automate this i.e. make the 's3 folder' available to the report.py script is non-trivial, and will need additional testing.

Reply:
This has been fixed in the current PR itself.